steal.lic

```diff
@@ -104,7 +115,7 @@ class Steal
   end
 
   def do_buff(buffs, settings)
-    buffs['spells'].each { |spell| cast_spell(spell, settings) }
+    buffs['spells'].each { |spell| buff(spell, settings) }
```

Comment:
the buff method is deprecated and going to be removed #2337

Reply:
OK, now it is removed :P

Reply:
Oops, ok, reverting this!  Was an unintentional reversion during merge.